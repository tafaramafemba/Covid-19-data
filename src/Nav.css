@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;1,100;1,200;1,300;1,400;1,500;1,600&family=Roboto+Slab:wght@900&display=swap');

nav {
  display: flex;
  flex-direction: row;
  background-color: #4369b2;
  justify-content: space-between;
  align-items: center;
}

.nav-title {
  font-family: 'Lato', sans-serif;
  color: #fff;
}

.nav-ul {
  display: flex;
  flex-direction: row;
  align-items: center;
  margin: 0;
}

li {
  text-decoration: none;
  list-style: none;
  padding-left: 10px;
  margin-right: 15px;
}

button {
  padding: 30px;
  background-color: #4369b2;
  border: none;
  background-image: url(/public/back.png);
  background-repeat: no-repeat;
  background-position: 10px;
}
