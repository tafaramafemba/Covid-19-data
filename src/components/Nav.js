import { Link } from 'react-router-dom';
import '../Nav.css';

function Nav() {
  return (
    <div>
      <nav>
        <Link to="/">
          <button className='back'></button>
        </Link>
        <h3 className='nav-title'>
            {window.location.pathname.split('/')[1]}
        </h3>
        <ul className='nav-ul'>
          <li className='audio'><img src='microphone.png' /></li>
          <li className='settings'><img src='settings.png' /></li>
        </ul>
      </nav>
    </div>
  );
}

export default Nav;