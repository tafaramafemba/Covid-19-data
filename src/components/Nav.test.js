import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from './Nav';

describe('Nav component', () => {

  test('component match with the snapshot', () => {
    const tree = renderer.create(<BrowserRouter><Nav /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});