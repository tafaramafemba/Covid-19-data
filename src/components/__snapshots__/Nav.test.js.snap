// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component component match with the snapshot 1`] = `
<div>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        className="back"
      />
    </a>
    <h3
      className="nav-title"
    />
    <ul
      className="nav-ul"
    >
      <li
        className="audio"
      >
        <img
          src="microphone.png"
        />
      </li>
      <li
        className="settings"
      >
        <img
          src="settings.png"
        />
      </li>
    </ul>
  </nav>
</div>
`;
